modules/plot/main/coffee/Axis.coffee

```diff
@@ -85,6 +85,8 @@ class Axis
     @xTitleHeight = 0
     @yAxisSize = 50
     @yTitleHeight = 0
+    options?.series?.forEach ([seriesType, seriesOpts]) =>
```

Comment:
Looks like series have to be provided as tuples? Perhaps objects would make more sense?
```
options.series = [
  {
    type: 'line',
    options: {...}
  },
  {
    type: 'bar',
    options: {...}
  }
]
```